### PR TITLE
Window manager cleanup

### PR DESCRIPTION
**Pull request by pekkaklarck**
New attempt to create a PR. Hopefully no conflicts this time.

### CODE REVIEW

File: test/unit/locators/test_windowmanager.py [hunk 1]
```diff
@@ -1,312 +1,240 @@
...
-        browser = mock()
+        manager = WindowManagerWithMockBrowser()
         with self.assertRaises(ValueError) as err:
-            manager.select(browser, "something=test1")
+            manager.select("something=test1")
             self.assertEqual(
                 str(err),
                 "Window locator with prefix 'something' is not supported"
             )
-        unstub()
```
Comment (aaltat) [inline test/unit/locators/test_windowmanager.py:20]: It might be good idea to call unstub() in `def tearDown`.

**Review by aaltat — COMMENTED** [review]
I had one comment about calling the `unstub`. But I need to read the the changes from bigger screen. Specially those unit test changes where hard to read from the mobile.

File: test/unit/locators/test_windowmanager.py (hunk 1 above)
Comment (pekkaklarck) [reply test/unit/locators/test_windowmanager.py:20]: Is it needed when we don't actually stub anything here? We just mock `ctx`.

Comment (aaltat) [reply test/unit/locators/test_windowmanager.py:20]: I did read the mockito docs again and you are right is it needed when doing stubs.

**Pull request merged**